synthesis/synth.tcl

```diff
@@ -52,6 +52,9 @@ yosys proc -nomux
 yosys proc_mux
 yosys flatten
 
+# Remove $print cells.
```

Comment:
A description of **_why_** `$print` cells are being removed would be good.

Reply:
Updated!